en/computer_vision/path_planning_interface.md

```diff
@@ -1,12 +1,18 @@
 # Path Planning Offboard Interface
```

Comment:
So I'm not sure if the Bezier curve works under the offboard interface. It might, but I'd be pleasantly surprised.

Reply:
OK, now I understand. This title was supposed to indicate that Path Planning is an interface driven by a companion computer, not that it is an interface that is active in offboard mode. I will just rename it Path Planning Interface to reduce this confusion.

Thanks